tests/SideBySide/Transaction.cs

```diff
@@ -50,6 +52,50 @@ public void DbConnectionCommit()
 			Assert.Equal(new[] { 1, 2 }, results);
 		}
 
+		[Theory]
+		[InlineData(IsolationLevel.ReadUncommitted, "read uncommitted")]
+		[InlineData(IsolationLevel.ReadCommitted, "read committed")]
+		[InlineData(IsolationLevel.RepeatableRead, "repeatable read")]
+		[InlineData(IsolationLevel.Serializable, "serializable")]
+		[InlineData(IsolationLevel.Snapshot, "repeatable read")]
+		[InlineData(IsolationLevel.Unspecified, "repeatable read")]
+		public void DbConnectionIsolationLevel(IsolationLevel inputIsolationLevel, string expectedTransactionIsolationLevel)
+		{
+			DbConnection connection = m_connection;
+			m_connection.Execute(@"set global log_output = 'table';");
+			m_connection.Execute(@"set global general_log = 1;");
```

Comment:
Thanks for adding tests for the new behaviour!

Since multiple tests can be run in parallel, is it possible that `set global general_log` is capturing log statements from other tests running simultaneously on other connections? You might be able to filter by `thread_id` to this connection's log output (using `MySqlConnection.ServerThread`).

Reply:
Thanks for the tip, will try it out

Reply:
Are you happy with this now the tests are passing?

Reply:
Sorry, I didn't see that the PR was updated and passing. Thanks for pinging me; I'll check it out soon.